src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionTest.cs

```diff
@@ -26,7 +26,7 @@ public void ConnectionTest()
 
 
         [PlatformSpecific(TestPlatforms.Windows)]  // Integ auth on Test server is supported on Windows right now
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))] // https://github.com/dotnet/corefx/issues/19218
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotArmProcessAndIsNotWindowsNanoServer))] // https://github.com/dotnet/corefx/issues/19218 And https://github.com/dotnet/corefx/issues/21598
```

Comment:
I just notice it's params: `public ConditionalFactAttribute(params string[] conditionMemberNames)`